queries/ruby/highlights.scm

```diff
@@ -45,7 +45,7 @@
  "while"
  ] @repeat
 
-(constant) @constant
+(constant) @type
```

Comment:
I think this should stay as `@constant` ?

Reply:
to be honest I'm still learning this query syntax, so I'm not sure. I applied the suggested change and it didn't changed the highlight, so I'm ok with it:

<img width="390" alt="_HOME_src_ruby-highlight_rb" src="https://user-images.githubusercontent.com/120483/102625124-ed4cab00-413c-11eb-8bc1-3ab9aa684bfc.png">

Reply:
Do you have a constant defined here (I am not a Ruby expert) ?

Reply:
Oh, thanks for asking, your question made me noticed that `::Some::High::Name::Space` is not being highlighted as type/constant. 😬 

In ruby anything all upcased is a constant, but types (stuff that starts with an Upcase) is usually also highlighted as a constant.

Reply:
I am not sure what you mean, so I guess I'd ask the opinion of @TravonteD :wink:

Reply:
If found this also strange when I updated the queries. Before the query was just never activated due to a syntax error.

tree-sitter-ruby unified `call` and `method_call`

Reply:
@theHamsta, sorry, it's not clear to me. Should I make any change on these queries?

Reply:
No worries. It just mean that there is only `call` now. `method_call` no longer exists.